src/shared/coredump-util.h

```diff
@@ -26,5 +26,15 @@ const char* coredump_filter_to_string(CoredumpFilter i) _const_;
 CoredumpFilter coredump_filter_from_string(const char *s) _pure_;
 int coredump_filter_mask_from_string(const char *s, uint64_t *ret);
 
+int parse_auxv(int log_level,
+               bool elf64bit,
```

Comment:
exposing this as a bool sounds a bit weird, no? why not just expose this as `uint8_t elfclass`, so that people pass ELFCLASS64 or ELFCLASS32 here? elf.h is part of glibc hence this should be a non-issue